download_files.py

```diff
@@ -10,14 +10,14 @@ def _fetch_urls_and_outpaths(file_path: Path) -> List[Tuple[str, str]]:
     urls_and_outpaths = []
     with open(file_path, "r") as fp:
         for line in fp:
-            tokens = tuple([t.strip() for t in line.split(',')])
+            tokens = [t.strip() for t in line.split(',')]
```

Comment:
what does this change do? why was it a tuple before?

Reply:
Oh I mostly just needed to convert the outpath to a Path variable. Before I could just tuple the token list to get Tuple[str, str], but now I needed Tuple[str, Path]